topic7/pom.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project
+  xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>netgloo</groupId>
```

Comment:
The groupId is generally unique amongst an organization or a project. The group acts much like the Java packaging structure does in an operating system. In this exercise if you had declared a package "com,mhaure" that should be the value to be configure here.